### PR TITLE
monitoring MVP

### PR DESCRIPTION
**Pull request by antdanchenko**
Added monitoring MVP for Laconic using Grafana and Prometheus

Please, try to run it using steps from laconic/monitoring-mvp/README.md and review
Would appreciate any comments around setup (first time using this tools)
Thanks in advance!

## Screenshot
Screenshot 2022-08-27 at 07 12 08

### CODE REVIEW

**Comment by Serhy** [comment]
I've tried on my machine from scratch. When running the readme i've got error:

```git checkout monitoring-mvp
Branch 'monitoring-mvp' set up to track remote branch 'monitoring-mvp' from 'origin'.
Switched to a new branch 'monitoring-mvp'
MacBook-Pro-2:laconic sergiichumak$ cd monitoring-mvp/
MacBook-Pro-2:monitoring-mvp sergiichumak$ docker-compose up -d
Creating network "monitoring-mvp_default" with the default driver
Pulling laconic (gatewayfm/laconic:main)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling laconic (gatewayfm/laconic:main)...
ERROR: pull access denied for gatewayfm/laconic, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
MacBook-Pro-2:monitoring-mvp sergiichumak$ 
```

File: monitoring-mvp/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+version: '2.2'
```
Comment (sibuser) [inline monitoring-mvp/docker-compose.yml]: Could we use latest version?

File: monitoring-mvp/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+services:
+
+  laconic:
+    image: gatewayfm/laconic:main
+    volumes:
+      - ./genesis.json:/root/.chibaclonkd/config/genesis.json
+      - ./config.toml:/root/.chibaclonkd/config/config.toml
+    command:
+        chibaclonkd start --gql-playground --gql-server
+    ports:
```
Comment (sibuser) [inline monitoring-mvp/docker-compose.yml:10]: If the port is not needed to be accessible from outside, you can use
```suggestion
    expose:
       - 
```

**Comment by sibuser** [comment]
Try again.


> I've tried on my machine from scratch. When running the readme i've got error:
> 
> 
> 
> ```git checkout monitoring-mvp
> 
> Branch 'monitoring-mvp' set up to track remote branch 'monitoring-mvp' from 'origin'.
> 
> Switched to a new branch 'monitoring-mvp'
> 
> MacBook-Pro-2:laconic sergiichumak$ cd monitoring-mvp/
> 
> MacBook-Pro-2:monitoring-mvp sergiichumak$ docker-compose up -d
> 
> Creating network "monitoring-mvp_default" with the default driver
> 
> Pulling laconic (gatewayfm/laconic:main)...
> 
> ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.
> 
> 
> 
> Continue with the new image? [yN]y
> 
> Pulling laconic (gatewayfm/laconic:main)...
> 
> ERROR: pull access denied for gatewayfm/laconic, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
> 
> MacBook-Pro-2:monitoring-mvp sergiichumak$ 
> 
> ```

**Comment by Serhy** [comment]
> Try again.
> 
> > I've tried on my machine from scratch. When running the readme i've got error:
> > ```
> > 
> > Branch 'monitoring-mvp' set up to track remote branch 'monitoring-mvp' from 'origin'.
> > 
> > Switched to a new branch 'monitoring-mvp'
> > 
> > MacBook-Pro-2:laconic sergiichumak$ cd monitoring-mvp/
> > 
> > MacBook-Pro-2:monitoring-mvp sergiichumak$ docker-compose up -d
> > 
> > Creating network "monitoring-mvp_default" with the default driver
> > 
> > Pulling laconic (gatewayfm/laconic:main)...
> > 
> > ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.
> > 
> > 
> > 
> > Continue with the new image? [yN]y
> > 
> > Pulling laconic (gatewayfm/laconic:main)...
> > 
> > ERROR: pull access denied for gatewayfm/laconic, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
> > 
> > MacBook-Pro-2:monitoring-mvp sergiichumak$ 
> > ```

Yes, it works now for me. Thank you, Alex!

**Comment by Serhy** [comment]
@antdanchenko I can successfully finish execution of `docker-compose up -d`

File: monitoring-mvp/docker-compose.yml (hunk 1 above)
Comment (zheli) [reply monitoring-mvp/docker-compose.yml]: `version` is actually deprecated (see https://docs.docker.com/compose/compose-file/#compose-file)

> The Compose file is a [YAML](http://yaml.org/) file defining [version](https://docs.docker.com/compose/compose-file/#version-top-level-element) (DEPRECATED), [services](https://docs.docker.com/compose/compose-file/#services-top-level-element) (REQUIRED), [networks](https://docs.docker.com/compose/compose-file/#networks-top-level-element), [volumes](https://docs.docker.com/compose/compose-file/#volumes-top-level-element), [configs](https://docs.docker.com/compose/compose-file/#configs-top-level-element) and [secrets](https://docs.docker.com/compose/compose-file/#secrets-top-level-element). The default path for a Compose file is compose.yaml (preferred) or compose.yml in working directory. Compose implementations SHOULD also support docker-compose.yaml and docker-compose.yml for backward compatibility. If both files exist, Compose implementations MUST prefer canonical compose.yaml one.

And it seems it doesn't do anything really. It's only informational for the coder.
> Top-level version property is defined by the specification for backward compatibility but is only informative.

> A Compose implementation SHOULD NOT use this version to select an exact schema to validate the Compose file, but prefer the most recent schema at the time it has been designed.

**Pull request merged**